config/samples/gcp/gcp_openshift_route.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: externaldns.olm.openshift.io/v1alpha1
+kind: ExternalDNS
+metadata:
+  name: sample-gcp
+spec:
+  domains:
+  - filterType: Include
+    matchType: Exact
+    name: misalunkgcp.qe.gcp.devcluster.openshift.com
+  provider:
+    type: GCP
+  source:
+    hostnameAnnotation: Allow
+    openshiftRouteOptions:
+      routerName: default
+    type: OpenShiftRoute
+  zones:
+  - <zone name>
```

Comment:
If you put a domain name similar to a real one, let's put a zone ID (for GCP it's zone ID not name) similar to a real one too, not just a placeholder.
And can you add a new line at the end of the line please?